contracts/contracts/staking/SingleAssetStaking.sol

```diff
@@ -188,7 +188,7 @@ contract SingleAssetStaking is Initializable, Governable {
 
         totalOutstanding = totalOutstanding.add(_totalExpected(newStake));
 
-        emit Staked(staker, amount);
+        emit Staked(staker, amount, duration, rate);
```

Comment:
rate is a uint240. But the Staked event uses a uin256. Does solidity automatically converts from uint240 to uint256 when emitting the event or do we need to cast?

Reply:
Yeah, solidity does and can uint240 to uint256 but the other way isn't possible. 

Let me know if this has to be changed to keep it consistent

Reply:
Thanks for checking. No need I think, to change as long as we make sure the event data is emitted correctly.